src/resolvers/Post/creator.ts

```diff
@@ -0,0 +1,8 @@
+import { PostResolvers } from "../../types/generatedGraphQLTypes";
+import { User } from "../../models";
+
+export const creator: PostResolvers["creator"] = async (parent) => {
+  return await User.find({
```

Comment:
`.find()` always returns an array, you have to use `findOne`, `findUnique` or something.